offchain/dispatcher/src/setup.rs

```diff
@@ -94,15 +94,21 @@ pub async fn create_context(
         .timestamp
         .as_u64();
     let epoch_length = config.epoch_duration;
+
+    let status = broker.status().await.context(BrokerSnafu)?;
+
+    // The dispatcher doesn't work properly if there are inputs in the broker from a previous run.
+    // Hence, we make sure that the broker is in a clean state before starting.
```

Comment:
I gave up trying to solve the bug because we should always restart Redis when the node restarts. Hence, the dispatcher will fail during startup when there is an input in Redis. This will be improved when we rewrite the node in Go.

Reply:
:/ what a bummer.

State all spread around is very annoying.